msg/GenericTrajectory.msg

```diff
@@ -0,0 +1,3 @@
+Header header
```

Comment:
Header here is redundant, as both `cartesian_trajectory` and `joint_trajectory` have one.